_docs/advanced/javascript.md

```diff
@@ -44,30 +42,8 @@ To limit a script to a given output format, use `site.output`:
 ```
 {% endraw %}
 
-## Adding scripts to epubs
-
-Scripts in epub are special. While web and PDF scripts should be in `/assets/js`, all scripts to be used in your epub should be in `_epub/js`. This is because you must not have any scripts in your epub that you aren't using, or it won't validate. By keeping epub scripts separate, only your epub scripts will end up in the epub package.
-
-Also, epub scripts must have a YAML frontmatter block, and use a layout with no HTML:
-
-``` md
----
-layout: min
----
-```
-
-This block tells Jekyll to process the script, which means it knows about it, which the epub packager needs in order to include it in the epub manifest.
+## Scripts in epub
 
-### Linking to scripts for epub
+All scripts to be used in your epub should be included in `assets/js/bundle/js`, and not added to pages as separate files. This is because you must not have any scripts in your epub that you aren't using, or it won't validate; and we only support including `bundle.js` in epub output.
```

Comment:
Should `assets/js/bundle/js` rather be `assets/js/bundle.js`?

Reply:
Thanks! Fixed in https://github.com/electricbookworks/electric-book/pull/540/commits/3e7b0bc3ab21f427225204f3a5408c4500613750.